openhub-ext/src/test/java/org/openhubframework/openhub/ri/AbstractIntegrationTest.java

```diff
@@ -0,0 +1,78 @@
+package org.openhubframework.openhub.ri;
+
+import static org.openhubframework.openhub.ri.AbstractIntegrationTest.STUB_SERVER_PORT;
+
+import com.xebialabs.restito.server.StubServer;
+import org.junit.After;
+import org.junit.Before;
+import org.openhubframework.openhub.test.TestConfig;
+import org.openhubframework.openhub.test.TestWebConfig;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.embedded.LocalServerPort;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.test.context.TestPropertySource;
+
+
+/**
+ * Simple parent for integration tests.
+ *
+ * @author Karel Kovarik
+ * @since 1.0.0
+ */
+@TestPropertySource(properties = {
+        "stub.server.port = " + STUB_SERVER_PORT,
+        "stub.server.url = localhost:${stub.server.port}",
+        "test.camel.initAllRoutes = true"
+})
+@SpringBootTest(
+        classes = {
+                TestConfig.class,
+                TestWebConfig.class
+        },
+        webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT
+)
+public abstract class AbstractIntegrationTest extends AbstractRITest {
```

Comment:
I would expect that parent class for web integration tests will be in OHF in test module, like AbstractTest or AbstractDbTest. It meens also add dependency to restito ...
Another project will have to creates identical/very similar classes, why don't add it to OHF?